application-admintools-default/src/main/java/com/xwiki/admintools/script/AdminToolsScriptService.java

```diff
@@ -172,13 +172,28 @@ public String getInstanceSizeSection() throws AccessDeniedException
      * @param order the order of the sort.
      * @return a {@link List} with the documents that have more than the given number of comments.
      */
-    public List<XWikiDocument> getPagesOverGivenNumberOfComments(long maxComments, Map<String, String> filters,
+    public List<DocumentReference> getPagesOverGivenNumberOfComments(long maxComments, Map<String, String> filters,
         String sortColumn, String order) throws AccessDeniedException
     {
         this.contextualAuthorizationManager.checkAccess(Right.ADMIN);
         return adminToolsManager.getPagesOverGivenNumberOfComments(maxComments, filters, sortColumn, order);
     }
 
+    /**
+     * Retrieve the empty documents from the XWiki instance.
+     *
+     * @param filters {@link Map} of filters to be applied on the gathered list.
+     * @param sortColumn target column to apply the sort on.
+     * @param order the order of the sort.
+     * @return a {@link List} with the empty documents.
+     */
+    public List<DocumentReference> getEmptyDocuments(Map<String, String> filters, String sortColumn, String order)
```

Comment:
Missing since and Unstable annotations, see https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#H40UnstableAnnotation